docs/source/release/v6.10.0/framework.rst

```diff
@@ -10,70 +10,128 @@ Algorithms
+- Algorithm :ref:`FindSXPeaks <algm-FindSXPeaks-v1>` supports new peak finding strategy ``AllPeaksNSigma``.
+  Credits to the author of SXD2001 for the idea of using NSigma as a threshold (albeit in SXD2001 the peak finding is done in 3D).
+  Gutmann, M. J. (2005). SXD2001. ISIS Facility, Rutherford Appleton Laboratory, Oxfordshire, England.
+- Algorithm :ref:`FindSXPeaks <algm-FindSXPeaks-v1>` now includes validation rules to remove spurious peaks due to noise,
+  by allowing user to provide additional arguements as below:
+  - ``MinNBinsPerPeak``, the Minimum number of bins contributing to a peak in an individual spectrum
+  - ``MinNSpectraPerPeak``, ``MaxNSpectraPerPeak`` Minimum & Maximum number of spectra contributing to a peak after they are grouped.
 
 Bugfixes
 ############
-.. amalgamate:: Framework/Algorithms/Bugfixes
+- Algorithm :ref:`LoadNexusProcessed <algm-LoadNexusProcessed>` is now faster to load a NeXus file.
+- Algorithm ``DSFinterp``, which was deprecated, has been removed
+- :ref:`algm-Abins` and :ref:`algm-Abins2D` no longer influence each other:
+
+  - :ref:`algm-Abins` algorithm sets the value
+    ``abins.parameters.sampling["bin_width"]`` while running. Previously this
+    overrided the default sampling of :ref:`algm-Abins2D` instruments if set.
+  - This did not cause results to be incorrect, but sampled
+    them on a different mesh to the expected one and could limit
+    resolution.
+  - Now the value is saved and restored after use by :ref:`algm-Abins`; it can
+    still be modified by users who wish to fiddle with the :ref:`algm-Abins2D`
+    behaviour.
+
+- Algortihm :ref:`Load <algm-Load>` now loads a single file faster.
+- Fixed doctest strings for :ref:`GenerateGroupingPowder <algm-GenerateGroupingPowder>`.
+- Algorithm :ref:`FindPeaks <algm-FindPeaks>` no longer crashes when the number of bins in the workspace are not sufficient to run :ref:`algm-SmoothData` algorithm.
 
 Fit Functions
 -------------
 
-New features
-############
-.. amalgamate:: Framework/Fit_Functions/New_features
-
 Bugfixes
 ############
-.. amalgamate:: Framework/Fit_Functions/Bugfixes
+- Search box for fitting functions in Fit interface no longer shows duplicate functions.
+- Fit Function DSFinterp1DFit, which was deprecated, has been removed.
+- Function :ref:`func-CompositeFunction` will now throw an exception if ``getNumberDomains()`` is called and there is an inconsistent number of domains in any of the member functions.
 
 
 Data Handling
 -------------
 
 New features
 ############
-.. amalgamate:: Framework/Data_Handling/New_features
+- Algorithm :ref:`algm-LoadEventAsWorkspace2D` accepts new boolean parameter ``LoadNexusInstrumentXML``. Default is *true*.
+- File search/loading will now look in instrument data cache **on IDAaaS**.
+  The instrument data cache is the directory ``/data/instrument/`` present on IDAaaS, and contains a local copy of the data archive.
```

Comment:
I don't think it's quite correct that it "contains a local copy of the data archive", is it just the most recent three years worth of data?

Reply:
Yeah, you're right, I will specify that

Reply:
I think you've got the same bullet point spacing error here as you had in the workbench docs, so probably something to do with the figure again.